examples/plugins/example_searchpath_plugin/setup.py

```diff
@@ -13,7 +13,9 @@
         long_description=LONG_DESC,
         long_description_content_type="text/markdown",
         url="https://github.com/facebookresearch/hydra/",
-        packages=find_namespace_packages(include=["hydra_plugins.*"]),
+        packages=find_namespace_packages(
+            include=["hydra_plugins.*", "arbitrary_package.*"]
+        ),
```

Comment:
I would say that the second package is technically not a namespace package.
Something like this (if works) is the correct form:
```
packages=find_namespace_packages(include=["hydra_plugins.*"]) + find_packages(include=["arbitrary_package.*"]),
```

Now, since this plugin is rather simple, you can also just add the second package directly.

```
packages=find_namespace_packages(include=["hydra_plugins.*"]) + ["arbitrary_package"],
```
(again, need to confirm it).

Reply:
thanks, fixed.